client/python/src/openlineage/client/transport/file.py

```diff
@@ -195,23 +204,45 @@ def _create_file_handler(self) -> FileHandler:
         else:
             return LocalFileHandler()
 
-    def _get_file_path(self) -> str:
+    def _get_file_path(self, event: Event) -> str:
         """Get the file path, adding timestamp if not in append mode."""
         if self.config.append:
             return self.config.log_file_path
-        else:
+        if not self.config.debug_mode:
             time_str = datetime.now().strftime("%Y%m%d-%H%M%S.%f")
             return f"{self.config.log_file_path}-{time_str}.json"
 
+        # Debug mode
+        time_str = datetime.now().strftime("%y%m%dT%H%M%S.%f")
```

Comment:
Why having different timestamp formats for debug and non-debug? The difference is only in year component 2025 -> 20, and using "T" as date-time delimiter